public/city.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>My City</title>
+    <link rel="stylesheet" href="/app.css">
+  </head>
+  <body>
+    <nav class="top-nav__city">
+      <div class="links">
+        <img class="logo" src="img/logo.svg">
+        <a class="name">Chris Duke</a>
+        <a href="/" class="skillz">My Skillz</a>
+        <a href="/city.html" class="city">My City</a>
+      </div>
+      <div class="intro">
+        <h1 class="student">Nashville, TN</h1>
+        <p class="details">Don't Expect Too Much - You'll Love It!</p>
+      </div>
+    </nav>
+
+    <div class="photos">
+      <div class="container">
+        <div class="nashville">
+          <h3>Nashville</h3>
+        </div>
+        <div class="top-row">
+          <img class="image sidewalk" src="img/sidewalk.jpg">
+          <img class="image skyline" src="img/skyline.jpg">
+        </div>
+        <div class="bottom-row">
+          <img class="image village" src="img/village.jpg">
+          <img class="image tunnel" src="img/tunnel.jpg">
+        </div>
+      </div>
+    </div>
+
+    <footer>
+      <div class="social">
+        <a class="icons icon-white" href="https://twitter.com/ChrisDanDuke"> <i class="fa fa-twitter" aria-hidden="true"></i></a>
+        <a class="icons icon-white" href="https://www.facebook.com/dukec"> <i class="fa fa-facebook" aria-hidden="true"></i></a>
+        <div class="logo"> <img class="logo" src="/logo.svg"> </div>
```

Comment:
I think the `src` for this should be `img/logo.svg` to match the image location within the public directory.